kettle/src/main/resources/standalone/configuration/hawkular-realm.json

```diff
@@ -57,17 +57,19 @@
       "secret" : "${uuid.hawkular.accounts.backend}"
     },
     {
-      "name": "hawkular-accounts-ui",
+      "name": "hawkular-ui",
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": true,
-      "secret": "${uuid.hawkular.accounts.ui}",
+      "secret": "${uuid.hawkular.ui}",
       "fullScopeAllowed" : false,
       "redirectUris": [
+        "http://localhost:8080/*",
```

Comment:
I'll check if there's a way to infer the port, but I'm afraid that would not be possible. What we can do is to add a set of ports that are "common" and add a documentation on how an administrator can add other addresses. 

In fact, having it "localhost" has already shown to be insufficient, as the continuous deployment server should be added to the list.